src/connectors/hits/connectHits.js

```diff
@@ -60,8 +60,21 @@ export default function connectHits(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.hits',
 
-      getConfiguration() {
-        return escapeHTML ? TAG_PLACEHOLDER : undefined;
+      getConfiguration(state) {
+        if (!escapeHTML) {
+          return state;
+        }
+
+        return state.setQueryParameters({
+          highlightPreTag:
+            state.highlightPreTag === undefined
+              ? TAG_PLACEHOLDER.highlightPreTag
```

Comment:
I feel like we should be always overriding it, but I'm not fully sure

Reply:
I removed it.